spec/system/admin/reports/sales_tax/sales_tax_totals_by_producer_spec.rb

```diff
@@ -119,6 +120,66 @@
     end
   end
 
+  context 'Order not to be shipped in a state affected by state tax rate' do
+    # Therefore, do not apply both tax rates here, only country one
+    before do
+      ship_address.update!({ state_id: another_state.id })
+      order.line_items.create({ variant:, quantity: 1, price: 100 })
+      order.update!({
+                      order_cycle_id: order_cycle.id,
+                      ship_address_id: ship_address.id
+                    })
+
+      while !order.completed?
+        break unless order.next!
+      end
```

Comment:
We have a service for that, you can use `OrderWorkflow.new(order).complete!`

Reply:
Did the replacement. While at it, I have also made the replacement for all other instances found in the spec.